jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/ChangeAccountSettingsActionGroupTest.kt

```diff
@@ -61,9 +59,8 @@ class ChangeAccountSettingsActionGroupTest {
             .filterIsInstance<ChangeRegionActionGroup>().first().getChildren(null)
             .filterIsInstance<ChangePartitionActionGroup>().first().getChildren(null)
 
-        val selectedAction = partitionActions.filterIsInstance<ToggleAction>().first()
-        assertNotNull(selectedAction)
-        assertThat(selectedAction.templateText).isEqualTo(selectedRegion.partitionId)
+        val selectedAction = partitionActions.firstOrNull { it.templateText == selectedRegion.partitionId }
```

Comment:
Probably reads better to say `.none {  }` and assert true

Reply:
good call.